test/recipes/80-test_ocsp.t

```diff
@@ -29,7 +30,12 @@ sub test_ocsp {
     my $title = shift;
     my $inputfile = shift;
     my $CAfile = shift;
+    my $untrusted = $CAfile;
     my $expected_exit = shift;
+    if (!looks_like_number($expected_exit)) {
```

Comment:
This looks quite horrible. Can we not just add the extra argument to the other tests (perhaps with "" meaning "use CAfile")?